final_prep/README.md

```diff
@@ -19,37 +19,41 @@ Some things to consider as you refactor include...
 
 Take your time as you go back and refactor your exercises from each section. We've included a handy checklist for you to go through below.
 
-- [ ] I have refactored my `section1` exercises to the best of my ability
-- [ ] I have refactored my `section2` exercises to the best of my ability
-- [ ] I have refactored my `section3` exercises to the best of my ability
-- [ ] I have refactored my `section4` exercises to the best of my ability
+- [x] I have refactored my `section1` exercises to the best of my ability
+- [x] I have refactored my `section2` exercises to the best of my ability
+- [x] I have refactored my `section3` exercises to the best of my ability
+- [x] I have refactored my `section4` exercises to the best of my ability
 
 ### Time Management Prep
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [x] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
+- [x] Study/Project work time
+- [x] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
+ ![Screen Shot 2021-12-18 at 12 53 09 PM](https://user-images.githubusercontent.com/95403666/146654074-ea261d98-a674-4dc4-8373-1959851c5c0a.png)
 - `Add Week 2 Screenshot Here`
+ ![Screen Shot 2021-12-18 at 12 53 16 PM](https://user-images.githubusercontent.com/95403666/146654086-0ca6a82e-0ff5-4616-afcd-965a3e67ae65.png)
 - `Add Week 3 Screenshot Here`
+ ![Screen Shot 2021-12-18 at 12 53 23 PM](https://user-images.githubusercontent.com/95403666/146654090-8ceb9cc1-9366-4962-bce4-f63b8ca6a0d0.png)
+ 
 
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] Add link to your gist here: https://gist.github.com/CoryBethune/271aa2e3e00afa1b252fc203411d6395
```

Comment:
Awesome!

RE your question about the "response times" questions - basically, if your mentor Slacks you, what time frame can you commit to getting back to them? It's easy to say "right away" but it's not realistic - you'll be in classes where you should be 100% engaged from 9-12 and 1-4, and sometimes EXHAUSTED after a long day. It's usually pretty standard to commit to "getting back to someone to acknowledge their message even if you can't fully answer" with 1/2 of a work day, and if you need more time to give a full response, doing so in 1 work day.

We mainly ask this question/push on this area because mentors sometimes have a hard time getting a hold of their mentee, and as a person who is volunteering their time, that can be particularly frustrating and it burns a bridge that might be... a really helpful bridge for the job hunt and beyond.